etc/p3-config/p3-config.yml

```diff
@@ -456,15 +491,15 @@ p3_image_mlnx_ofed:
     - "dhcp-all-interfaces"
     - "vm"
   env:
-    DIB_MLNX_OFED_VERSION: "4.0-2"
+    DIB_MLNX_OFED_VERSION: "4.1-1"
     DIB_MLNX_OFED_REPO: "{{ p3_repo_server_url_mlnx_ofed }}"
     DIB_MLNX_OFED_DELETE_REPO: "y"
     DIB_ALASKA_REPO: "{{ p3_repo_server_url_alaska_extras }}"
     DIB_ALASKA_PKGLIST: "pam-python pam-keystone"
     DIB_ALASKA_DELETE_REPO: "y"
   properties:
     os_distro: "centos"
-    os_version: 7
+    os_version: 7.4
```

Comment:
Hmm, we are note being too consistent on 7 vs 7.x if we change that here. I guess the build will some day just jump to 7.5 without us syncing that up?

Makes me wonder about those names.

Reply:
I think you're right about the version numbers, whether to add a minor version or not.  I had in mind that '7' would mean 'Latest CentOS 7' and '7.3' would be a specific versioned build.  But that's probably a bit vague.  Perhaps I'll back this out and keep it to an integer version in the property for now.

Reply:
What if we had os_version=7 and os_release=7.4 for CentOS and os_version=16 and os_release=16.04 for Ubuntu?

Reply:
...perhaps those terms should be the other way round.

Reply:
so... had a look at:
https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html

It seems we should actually move to 7.4 for the os_version here.

Maybe what we should do is actually pin these on specific base image version, rather than just pulling the latest, and have a script that can bump those URLs for us, as needed? That feels over complicated, but it means each time to generate the image using the definition in here, we have a better confidence it will be the same thing until we change something?

Something tells me I am overthinking all this... must be an easier way.